src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { TodoItem } from '../TodoItem/TodoItem';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  deleteTodo: (todoId: number) => Promise<void>,
+  updateTodoStatus: (todo: Todo, status: boolean) => Promise<void>,
+  updateTodoTitle: (todo: Todo, title: string) => Promise<void>,
+};
+
+export const TodoList: React.FC<Props> = ({
+  todos,
+  deleteTodo,
+  updateTodoStatus,
+  updateTodoTitle,
+}) => {
+  return (
+    <ul className="todoapp__main" data-cy="TodoList">
+      {todos.map(todo => {
+        return (
+          <TodoItem
+            key={todo.id}
+            todo={todo}
+            deleteTodo={deleteTodo}
+            updateTodoStatus={updateTodoStatus}
+            updateTodoTitle={updateTodoTitle}
+          />
+        );
+      })}
```

Comment:
You can immideately return the component like that:
```suggestion
      {todos.map(todo => (
          <TodoItem
            key={todo.id}
            todo={todo}
            deleteTodo={deleteTodo}
            updateTodoStatus={updateTodoStatus}
            updateTodoTitle={updateTodoTitle}
          />
       ))}
```